website/docs/r/aws_auth_backend_client.html.md

```diff
@@ -103,6 +107,18 @@ The following arguments are supported:
 * `max_retries` - (Optional) Number of max retries the client should use for recoverable errors. 
     The default `-1` falls back to the AWS SDK's default behavior.
 
+* `rotation_period` - (Optional) The amount of time in seconds Vault should wait before rotating the root credential.
+    A zero value tells Vault not to rotate the root credential. The minimum rotation period is 10 seconds. Requires Vault Enterprise 1.19+.
+
+* `rotation_schedule` - (Optional) The schedule, in [cron-style time format](https://en.wikipedia.org/wiki/Cron),
```

Comment:
Do we want something here about mutual exclusivity of rotation_window vs. period?

Reply:
Good question! Decided to leave that out since strictly speaking, TFVP does not enforce any mutual exclusivity on the fields. It is standard practice to let Vault enforce its mutual exclusivity. It would be technically possible to set both via TFVP, but Vault would return an error so the flow would fail and let the user know